app/src/main/res/layout/activity_qrexchange.xml

```diff
@@ -14,41 +14,80 @@
         android:fitsSystemWindows="true"
         android:theme="@style/AppTheme.AppBarOverlay">
 
-        <FrameLayout
+        <android.support.design.widget.AppBarLayout
+            android:id="@+id/app_bar_change_contact"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:orientation="horizontal">
+            android:fitsSystemWindows="true"
+            android:theme="@style/AppTheme.AppBarOverlay">
 
-            <TextView
-                android:id="@+id/toolbar_title"
+            <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
-                android:textAlignment="center"
-                android:gravity="center"
-                android:text="@string/app_name"
-                android:textSize="@dimen/toolbar_title_size"
-                android:textStyle="bold" />
-            <ImageButton
-                android:id="@+id/back_button_QR_exchange"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:onClick="onBackButtonClick"
-                android:src="?attr/actionModeCloseDrawable"
-                android:contentDescription="@string/go_back" />
-        </FrameLayout>
+                android:orientation="horizontal">
+
+                <ImageButton
+                    android:id="@+id/back_button_QR_exchange"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center"
+                    android:layout_weight="1"
+                    android:backgroundTint="@color/color_primary"
+                    android:contentDescription="@string/go_back"
+                    android:onClick="onBackButtonClick"
+                    tools:targetApi="lollipop"
+                    android:src="?attr/actionModeCloseDrawable" />
+
+                <TextView
+                    android:id="@+id/toolbar_title"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center"
+                    android:layout_weight="1"
+                    android:text="@string/app_name"
+                    android:textAlignment="center"
+                    android:textSize="@dimen/toolbar_title_size"
+                    android:textStyle="bold" />
+
+                <ImageButton
+                    android:id="@+id/empty_button"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center"
+                    android:layout_weight="1.5"
+                    android:backgroundTint="@color/color_primary"
+                    tools:targetApi="lollipop"
```

Comment:
@ericcornelissen @kcallebaut  @clenrock @TheBonheurs I think it may be better to upgrade our minimumSdkLevel to 21, that is, lollipop, because we're using tools:targetApi a bit too often. Most people, especially in our target group, have a somewhat recent device, so it shouldn't be a problem imo.